src/main/webapp/app/exercises/programming/manage/update/update-components/custom-build-plans/programming-exercise-custom-build-plan.component.ts

```diff
@@ -42,7 +43,7 @@ export class ProgrammingExerciseCustomBuildPlanComponent implements OnChanges {
     ngOnChanges(changes: SimpleChanges) {
         if (changes.programmingExerciseCreationConfig || changes.programmingExercise) {
             if (this.shouldReloadTemplate()) {
-                this.loadAeolusTemplate();
+                this.loadAeolusTemplate(changes.programmingExerciseCreationConfig.currentValue.isImportFromFile);
```

Comment:
This looks strage, why not just `this.isImportFromFile`?

Reply:
`this.isImportFromFile` isn't set yet  in this step, so it would always be false. We set it inside the `loadAeolusTemplate` method